third_party/pub/batch_compare_licenses.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+Future<void> main(List<String> args) async {
```

Comment:
Can we add a help comment that says what it does and prints? or maybe a `--help` handler?